fuzz/fuzz_minify.cpp

```diff
@@ -1,5 +1,8 @@
-#include "simdjson/jsonminifier.h"
-#include "simdjson/jsonparser.h"
+#include "simdjson.h"
+<<<<<<< HEAD
```

Comment:
Something is off here.

Reply:
... and I'm wondering how it compiled, ran and succeeded. Yeah, I'll recheck for merge conflicts, thanks